regtest/funnel/rt-funnel/config

```diff
@@ -0,0 +1,4 @@
+plumed_modules=funnel
+mpiprocs=1
+type=driver
+arg="--plumed plumed.dat --mf_trr small.trr --pdb start.pdb --dump-forces forces --dump-forces-fmt=%10.6f"
```

Comment:
@h2nch2co2h I think that it is sufficient to change the `--dump-forces-fmt` so as to use less digits (e.g. to `%9.5f`).